### ISSUE TITLE
Future of java-diff-utils

### ISSUE THREAD

**Issue by koritakoa** [java-diff-utils/java-diff-utils-jycr#11]
Sorry to raise this here, this is not specifically about this repo, more about what you are trying to achieve with the organization and the 3 repos. If there is a better way to raise issues about the organisation, please let me know.

Maybe there should be an org repo with github pages and a readme about the org?

It's clear you are trying to get a unified version of the library.

It's not clear what the status of that is. There are 3 repos in the org, which should I use? I picked this one for the issue because it had the most recent commit.

I see there are about 4 or 5 different different copies of this library on Github. Have you invited these developers to join the org? We don't want https://xkcd.com/927/ . Best bet is to try get as many invested parties working on the same repository, as I'm sure you're trying to do.

So basically, what is the status? Could we draw up a project plan and some information about the status of this project. I'd be willing to help with this.

**Comment by koppor** [comment]
I am very aware that there are many forks out there. Therefore, I searched the available forks and checked, which one is the most updated one. It is the one by @wumpz. I could convince him to move to here -> https://github.com/java-diff-utils/java-diff-utils. We will publish to maven central soon. Just having issues with access rights.

I updated the README.md from https://github.com/java-diff-utils/java-diff-utils-jycr accordingly. This repositor will be read-only soon to let all people know there to look at.

Finally, we are intending to publish a gh-pages site. Similar to <https://adr.github.io/madr/>. Maybe, we will use https://readthedocs.org/. Any documentation help is welcome!

**Comment by koppor** [comment]
After some googling, I found [“GitHub Stars” is a very useful metric. But for *what*?](https://opensource.stackexchange.com/q/5110/5671). According to research, it is a useful metric for measuring "engineered software projects". Thus, I when investigating forks, I would first investigate the forks with the highest number of stars.

To find maintained forks, http://forked.yannick.io/ is a useful tool. Example: http://forked.yannick.io/java-diff-utils/java-diff-utils. I am aware, that the tooling does not work if forks were not done using GitHub fork technology. However, in most settings, it helped me.
